File: java/fastcampus-java-spring/part3/chapter04/delete/src/main/java/com/example/delete/controller/DeleteApiController.java
package com.example.delete.controller;

import org.springframework.web.bind.annotation.*;

@RestController
@RequestMapping("/api")
public class DeleteApiController {
    @DeleteMapping("/delete/{userId}")
    public void delete(@PathVariable String userId, @RequestParam String account) {
        System.out.println(userId);
        System.out.println(account);
    }
}

File: spring/inflearn-steven-kotlin-springboot-webmvc/mvc/src/main/kotlin/com/example/mvc/controller/delete/DeleteApiController.kt
package com.example.mvc.controller.delete

import org.springframework.web.bind.annotation.*

@RestController
@RequestMapping("/api")
class DeleteApiController {

    @DeleteMapping(path=["/delete-mapping"])
    fun deleteMapping(
        @RequestParam(value="name") _name: String,
        @RequestParam(name="age") _age: Int
    ): String {
        println(_name)
        println(_age)
        return _name + " " + _age
    }

    @DeleteMapping(path=["/delete-mapping/name/{name}/age/{age}"])
    fun deleteMappingPath(@PathVariable(value="name") _name: String, @PathVariable(name="age") _age: Int): String {
        println(_name)
        println(_age)
        return _name + " " + _age
    }
}